docs/admin.md

```diff
@@ -0,0 +1,88 @@
+- Preferences/restrictions for Backstage and|or PostgreSQL images due to  Airgapped environment.
+- Existed Taints and tolerations policy, so Backstage Pods have to be configured with certain tolerations restrictions.
+- ...
+
+Default Configuration is implemented as a ConfigMap called *backstage-default-config*, deployed on *backstage-system* namespace and mounted to Backstage controller container as a */default-config* directory.
+This config map contains the set of keys/values which maps to file names/contents in the */default-config*.
+These files contain yaml manifests of objects used by Backstage controller as an initial desired state of Backstage CR according to Backstage Operator data model.
+(TODO: link to the diagram here) 
+
+Mapping of configMap keys (yaml files) to runtime objects (NOTE: for the time (Dec 20'23) it is a subject of change):
+| Key/File name         | k8s/OCP Kind        | Mandatory*    |Notes                                      |
+| ----------------------|:-------------------:| --------------|------------------------------------------:|
+| deployment.yaml       | appsv1.Deployment   | Yes           | Backstage deployment |
+| service.yaml          | corev1.Service      | Yes           | Backstage Service |
+| db-statefulset.yaml   | appsv1.Statefulset  | For DB enabled| PostgreSQL statefulSet    |    
+| db-service.yaml       | corev1.Service      | For DB enabled| PostgreSQL Service   |
+| db-secret.yaml        | corev1.Secret       | For DB enabled| Secret to connect Backstage to PSQL   |
+| route.yaml            | openshift.Route     | No (for OCP)  | Route exposing Backstage service    |
+| app-config.yaml       | corev1.ConfigMap    | No            | Backstage app-config.yaml    |
+| configmap-files.yaml  | corev1.ConfigMap    | No            | Backstage config file inclusions from configMap   |
+| configmap-envs.yaml   | corev1.ConfigMap    | No            | Backstage env variables from configMap    |
+| secret-files.yaml     | corev1.Secret       | No            | Backstage config file inclusions from Secret   |
+| secret-envs.yaml      | corev1.Secret       | No            | Backstage env variables from Secret    |
+| dynamic-plugins.yaml  | corev1.ConfigMap    | No            | dynamc-plugins config *    |
+
+
+NOTES: 
+ - Mandatory means it is needed to be present in either (or both) Default and CR Raw Configuration.
+ - dynamic-plugins.yaml is a fragment of app-config.yaml provided with RHDH/Janus-IDP. The reason it mentioned separately is specific way it provided (via mounting to dedicated initContainer)  
+
+### Operator Bundle configuration 
+
+With Backstage Operator's Makefile you can generate bundle descriptor using *make bundle* command
+
+Along with CSV manifest it generates default-config ConfigMap manifest, which can be modified and applied to Backstage Operator
+TODO: is kubectl sufficient or there are some other tools?
+
+### Kustomize deploy configuration
```

Comment:
intentional spelling here? is there an openshift equivalent tool for downstreaming this text, like using `oc` instead of `kustomize` ?

Reply:
Nothing I am aware about. Please let me know if there is an equvalent.

Reply:
so running `make deploy` will run `kustomize` ?

Reply:
yes